notebooks/en/vector_search_with_hub_as_backend.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>"One of DuckDB's features is vector similarity search..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/276/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>